playground/update.sh

```diff
@@ -20,18 +20,15 @@ gopherjs build -m
 # The GOPATH workspace where the GopherJS project is.
 gopherjsgopath=$(go list -f '{{.Root}}' github.com/gopherjs/gopherjs)
 
-rm -r pkg/
+rm -rf pkg/
```

Comment:
If the previous `go generate` failed before `pkg` was recreated this step fails otherwise.